var/spack/repos/builtin/packages/cgdb/package.py

```diff
@@ -17,6 +17,12 @@ class Cgdb(AutotoolsPackage):
     # Required dependency
     depends_on('ncurses')
     depends_on('readline')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
```

Comment:
I wonder if these dependencies are actually needed to build from a release, the tarball already contains a configure script.

Reply:
I was able to read the spack manual and understand what was pointed out.